client/deis.py

```diff
@@ -789,8 +790,11 @@ def auth_login(self, args):
             provide a username for the account.
           --password=<password>
             provide a password for the account.
+          --ssl-verify=false
+            disables ssl certificate verification
```

Comment:
Maybe something a bit more clear on the intent: `disables SSL certificate verification for API requests.`